README.md

```diff
@@ -5,6 +5,11 @@
 
 _Browse like no one’s watching. The new Firefox Focus automatically blocks a wide range of online trackers — from the moment you launch it to the second you leave it. Easily erase your history, passwords and cookies, so you won’t get followed by things like unwanted ads._
 
+<a href="https://f-droid.org/packages/org.mozilla.klar/" target="_blank">
+<img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="90"/></a>
```

Comment:
F-Droid contains only FLOSS apps, yours is welcome, but if you want me to remove this badge, I'll do ;)